src/java/com/vaklinov/zcashui/ZCashClientCaller.java

```diff
@@ -1337,9 +1337,10 @@ public boolean applyZendCommands() {
 				commands = config_file.getProperty("ZendCommands");
 				run_option = config_file.getProperty("RunOnce").trim().equals("1") ? true : false;
 
-				if (run_option) {
-					Log.info("RunOnce config file: " + run_option);
-					setConfig("", "1");
+				if (run_option && commandsFile.delete()) {
+					Log.info("Config deleted successfully");
+				}else{
+					Log.error("Config not deleted");
```

Comment:
Miner NIT: I'd like to see more descriptive error messages, `Advanced Panel config file not properly deleted.` would work well.  Capturing the error itself and logging that would be best, if it would be easy to implement...

Non-blocking NIT.